index.css

```diff
@@ -966,3 +966,83 @@ a:hover {
 .button:disabled {
   background-color: white;
 }
+
+.form-cont {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 58px;
+  position: absolute;
+  width: 375px;
+  height: 844px;
+  left: 0;
+  top: 6038px;
+}
+
+.h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin-bottom: 40px;
+  text-align: center;
+  color: #172b4d;
+}
+
+input {
+  border: none;
+  width: 327px;
+  height: 48px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  left: 30px;
+  padding-left: 10px;
+  display: flex;
+  align-items: center;
+}
+
+textarea {
+  border: none;
+  width: 317px;
+  height: 114px;
+  position: absolute;
+  margin-top: 10px;
+  padding: 10px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  background: #fbf8f7;
+  color: #3c3a39;
+}
+
+.line {
+  width: 327px;
+  height: 1px;
+  margin: 0;
+  background: #dbd8d7;
+}
+
+.get-in {
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover and active (when it's pressed), as shown in the Figma template below:

<img width="541" alt="image" src="https://user-images.githubusercontent.com/52934128/193900851-3cb52cfc-3deb-4bfa-9ad8-054f39c6ea76.png">

the button on the pressed state has the `background: #E05304` and on hover has the `background: #FFF5E1`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #E05304;
}
```
**for hover state**
```
button:hover {
    background-color: #FFF5E1;
}
```